android/core/build.gradle

```
@@ -41,30 +41,33 @@ android {
 }
 
 dependencies {
-    implementation(platform("com.squareup.okhttp3:okhttp-bom:4.11.0"))
 
     // For as long as we support API 25; once we can raise support to 26, all is fine
-    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.4'
+    coreLibraryDesugaring libs.desugar.jdk.libs
 
-    implementation 'androidx.core:core-ktx:1.13.1'
-    implementation 'androidx.appcompat:appcompat:1.7.0'
-    implementation 'com.squareup.okhttp3:okhttp'
-    implementation 'com.squareup.moshi:moshi:1.15.1'
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.1'
-    implementation 'org.jetbrains.kotlinx:kotlinx-datetime:0.6.0'
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.8.3'
+    implementation libs.androidx.ktx
+    implementation libs.androidx.appcompat
 
+    implementation libs.kotlinx.coroutines
+    implementation libs.kotlinx.datetime
+    implementation libs.androidx.lifecycle.viewmodel.ktx
+
+    implementation(platform(libs.okhttp.bom))
+    implementation(libs.okhttp.core)
+    implementation libs.moshi
+
+    // TODO: Fix this. Maybe related?: https://github.com/gradle/gradle/issues/21267
+    //noinspection UseTomlInstead
```

Ooh interesting. I'm just going to suggest making the comment a bit clearer so it's obvious what's broken.

```suggestion
    // TODO: Migrate version to TOML (doesn't work). Likely related issue: https://github.com/gradle/gradle/issues/21267
    //noinspection UseTomlInstead
```